archive/conf/routes

```diff
@@ -7,5 +7,7 @@ GET        /_healthcheck                            controllers.HealthCheck.heal
 
 GET        /dev/ui                                  controllers.DevComponentController.renderComponent()
 
+GET        /dev/props                               controllers.DevComponentController.props()
```

Comment:
To keep convention with other routes we have which return json maybe lets have something like:
```
`dev/props.json`
```